src/spec-common/injectHeadless.ts

```diff
@@ -125,6 +126,37 @@ export interface CommonDevContainerConfig {
 	userEnvProbe?: UserEnvProbe;
 }
 
+export interface CommonContainerMetadata {
```

Comment:
For my future reference, what does the `Common` prefix signify?

Could this interface be included in `CommonDevContainerConfig`?

Reply:
`common` is for "any container engine" (sources are in a `spec-common` folder). Maybe not the best naming, I introduced it when we added support for attaching to k8s containers in the extension. Currently this distinction is not really used in the CLI, but we might revive it in the future.